typescript/sdk/src/gas/HyperlaneIgpChecker.ts

```diff
@@ -89,7 +90,9 @@ export class HyperlaneIgpChecker extends HyperlaneAppChecker<
         expectedOverhead = 0;
       }
 
-      const remoteId = this.multiProvider.getDomainId(remote);
+      const remoteId =
```

Comment:
can you leave a comment "in case we don't have the chain in core chain metadata (like for CLI), we need the optional ? part here"